components/src/store/index.ts

```diff
@@ -26,11 +26,11 @@ export const configureStore = () => {
         console.error("Failed to read pulumi_state from localStorage:", e);
     }
 
-    const persistedState: Partial<AppState> = local ? JSON.parse(local) : {};
+    const persistedState: any = local ? JSON.parse(local) : {};
```

Comment:
I'm not a huge fan of using `any` but knowing you, I'm sure is a good reason to have to use it here.

Reply:
Yeah, that's because this is literally "whatever's in localStorage", so we can't rely on that being of any particular shape at the point of deserialization, since you can throw just about anything in there. 😄 Good spot, though.